connector/charts/oaebudt-connector/values.yaml

```diff
@@ -188,9 +198,17 @@ service:
 
 # This block is for setting up the ingress for more information can be found here: https://kubernetes.io/docs/concepts/services-networking/ingress/
 ingress:
-  enabled: enable
-  className: ""
-  annotations: []
+  enabled: true
+  className: "alb"
+  annotations:
+    internet-facingalb.ingress.kubernetes.io/certificate-arn: arn:aws:acm:us-east-1:605134435349:certificate/f5f64987-094b-4de6-9c8c-d81beb7014f7
```

Comment:
if set-up correctly, you don't have to hardcode the certificate arn, ALB ingress controller can figure it out based on the hostname